apis/v1alpha1/gatewayclass_types.go

```diff
@@ -103,7 +103,7 @@ type GatewayClassSpec struct {
 // resource for a GatewayClass.
 //
 // +k8s:deepcopy-gen=false
-type GatewayClassParametersObjectReference = LocalObjectReference
+type GatewayClassParametersObjectReference LocalObjectReference
```

Comment:
Is there any point in leaving these aliases in place or should we just use `LocalObjectReference` directly? I think we're already doing that for a lot of references in the API.

Reply:
Yeh, just using `LocalObjectReference` directly makes it easier to read IMHO

Reply:
Don't really have a preference, I can change it.